File: desafio-02/juscelinodjj/javascript/desafio-02.js
// Escreva um programa para listar todos os números primos entre 1 e 10000
const getPrimeNumbers = range => {
    return (isNaN(range) || range < 2)
        ?   "Insira um número válido, sendo ele maior ou igual a dois"
        :   Array(parseInt(range)).fill(null).map((_, index) => ++index)
                .filter( number => {
                    const dividers = Array(number).fill(null).map((_, index) => ++index);
                    const dividersForThisNumber = dividers.filter( divider => number % divider === 0);
                    return dividersForThisNumber.length === 2 ? number : null;
                });
};
console.log(getPrimeNumbers(10000));

File: desafio-02/FagnerDSantos/desafio_02.py
## ENCONTRAR NÚMEROS PRIMOS DE 1 A 1000

for num in range (2, 1000):
    num_primos = True
    
    for divisor in range (2, num):
        if num % divisor == 0:
            num_primos = False
    
    if num_primos:
        print (num)